packages/livekit-server-sdk/src/SipClient.ts

```diff
@@ -166,11 +166,15 @@ export class SipClient extends ServiceBase {
    * @param name - human-readable name of the trunk
    * @param numbers - phone numbers of the trunk
    * @param opts - CreateSipTrunkOptions
+   * @param headers - SIP X-* headers to be included in 200 OK responses (optional)
+   * @param headersToAttributes - map of SIP X-* headers from INVITE to SIP participant attributes (optional)
    */
   async createSipInboundTrunk(
     name: string,
     numbers: string[],
     opts?: CreateSipInboundTrunkOptions,
+    headers?: { [key: string]: string },
```

Comment:
these should be included in opts

Reply:
oh okay, was debating whether these should be part of options. will update now